### PR TITLE
Remove edit permissions in Che namespace

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,32 @@ cd evals/
...
+Before running the playbook, create a new OAuth Application on GitHub. This can
+be done at https://github.com/settings/developers. Note the `Client ID` and
+`Client Secret` fields of the OAuth Application, these are required by the Che
+playbook.
+
+Set the following variables:
+
+* `che_github_client_id` - The `Client ID` of the created GitHub OAuth Application.
+* `che_github_client_secret` - The `Client Secret` of the created GitHub OAuth Application.
+* `che_route_suffix` - The router suffix of the OpenShift cluster.
```
Comment (pmccarthy) [inline README.md]: @aidenkeating Not directly related to this PR but if the ```che_route_suffix``` is similar to the ```evals_app_host``` var that was added as part of https://github.com/integr8ly/installation/pull/14 then we should be able to dynamically set the router suffix? See https://github.com/integr8ly/installation/blob/7f4a87e63e194aada112662d557c247350fd330e/evals/playbooks/install-all.yml#L47

The less vars that need to be set the better 😃

**Comment by pmccarthy** [comment]
@aidenkeating Since we're updating the README here, would you mind adding Che to the list of products in the overview section please? https://github.com/integr8ly/installation/blob/master/README.md#overview

File: README.md [hunk 2]
```diff
@@ -89,7 +89,32 @@ cd evals/
...
+* `che_keycloak_user` - Username to authenticate as, this would be the admin user by default.
+* `che_keycloak_password` - Password of the user.
+* `che_namespace` - The namesapce to provision che into.
+* `che_infra_namespace` - This can usually be the same as `che_namespace`.
+
+Run the playbook.
+
+```shell
+cd evals/
+ansible-playbook -i inventories/hosts playbooks/che.yml
```
Comment (pmccarthy) [inline README.md]: The playbook is named ```che-install.yml``` rather than ```che.yml```

File: README.md (hunk 1 above)
Comment (aidenkeating) [reply README.md]: @pmccarthy Yep that makes sense! I'll update the defaults to use the variable if it's available. That can be used by the `install-all.yml` playbook in #14. This documentation is for running the Che playbook independently so I'll keep the documentation here for now.

**Comment by pmccarthy** [comment]
@aidenkeating The removal of the user edit permissions for the che namespace and the update to Che docs made it back to master as part of https://github.com/integr8ly/installation/pull/14

I also noticed that the merged README mentioned Che and Launcher but didn't have them listed so I went ahead and issued a separate PR for that: https://github.com/integr8ly/installation/pull/20

So for this PR, the only thing needed at this point is setting the ```che_route_suffix ``` var in defaults

**Pull request merged**